cdap-api-spark2_2.11/pom.xml

```diff
@@ -53,11 +53,23 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_2.11</artifactId>
       <scope>provided</scope>
+      <exclusions>
```

Comment:
Are we excluding it everywhere? If so, please do it with dependency management instead of adding it everywhere. Are you sure it's not used at all in all scenarios?
If we need a different version, please use dependencyManagement as well instead of exclusions.